config/config_test.go

```diff
@@ -378,6 +401,37 @@ func TestFullConfig(t *testing.T) {
 		cmpBools(t, "cfg.BlacklistedAppMap", cfg.BlacklistedAppMap[cfg.BlacklistedApps[i]], true)
 	}
 
+	//Assert purpose VendorExceptionMap hash tables were built correctly
+	assert.Equal(t, 1, len(cfg.GDPR.TCF2.Purpose1.VendorExceptionMap), "cfg.GDPR.TCF2.Purpose1.VendorExceptionMap length")
+	assert.Equal(t, 1, len(cfg.GDPR.TCF2.Purpose2.VendorExceptionMap), "cfg.GDPR.TCF2.Purpose2.VendorExceptionMap length")
+	assert.Equal(t, 1, len(cfg.GDPR.TCF2.Purpose3.VendorExceptionMap), "cfg.GDPR.TCF2.Purpose3.VendorExceptionMap length")
+	assert.Equal(t, 1, len(cfg.GDPR.TCF2.Purpose4.VendorExceptionMap), "cfg.GDPR.TCF2.Purpose4.VendorExceptionMap length")
+	assert.Equal(t, 4, len(cfg.GDPR.TCF2.Purpose5.VendorExceptionMap), "cfg.GDPR.TCF2.Purpose5.VendorExceptionMap length")
+	assert.Equal(t, 1, len(cfg.GDPR.TCF2.Purpose6.VendorExceptionMap), "cfg.GDPR.TCF2.Purpose6.VendorExceptionMap length")
+	assert.Equal(t, 1, len(cfg.GDPR.TCF2.Purpose7.VendorExceptionMap), "cfg.GDPR.TCF2.Purpose7.VendorExceptionMap length")
+	assert.Equal(t, 1, len(cfg.GDPR.TCF2.Purpose8.VendorExceptionMap), "cfg.GDPR.TCF2.Purpose8.VendorExceptionMap length")
+	assert.Equal(t, 1, len(cfg.GDPR.TCF2.Purpose9.VendorExceptionMap), "cfg.GDPR.TCF2.Purpose9.VendorExceptionMap length")
+	assert.Equal(t, 4, len(cfg.GDPR.TCF2.Purpose10.VendorExceptionMap), "cfg.GDPR.TCF2.Purpose10.VendorExceptionMap length")
```

Comment:
Gotcha I think I have the coverage because I am testing each map through a length check plus a check for specific keys in the map just after the length check but I should look for a single testify map compare function to use instead.

Reply:
Ah, ok. Yes. I think the coverage is there. Perhaps it's just that an object compare would be more straight forward.